simplified-ui-accounts/src/main/res/layout/account.xml

```diff
@@ -274,18 +274,32 @@
 
       <TextView
         android:id="@+id/accountReportIssueText"
+        android:background="?android:attr/selectableItemBackground"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="16dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="16dp"
         android:layout_marginBottom="16dp"
         android:text="@string/accountReportIssue"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/accountReportIssueEmail"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+      <TextView
+        android:id="@+id/accountReportIssueEmail"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:fontFamily="monospace"
+        android:text="someone@example.com"
```

Comment:
Please use `tools:text` here. Any attributes with this namespace are stripped on compile.

Reply:
What're the practical consequences of that? Is that just so that the `someone@example.com` string doesn't actually appear in the app, but _does_ appear in the layout editor?

Reply:
> Is that just so that the `someone@example.com` string doesn't actually appear in the app, but does appear in the layout editor? &mdash; @io7m 

Yep! That's exactly right.

https://developer.android.com/studio/write/tool-attributes